molgenis-data-import/src/main/java/org/molgenis/data/importer/emx/exception/EmxException.java

```diff
@@ -0,0 +1,11 @@
+package org.molgenis.data.importer.emx.exception;
+
+import org.molgenis.i18n.CodedRuntimeException;
+
+public abstract class EmxException extends CodedRuntimeException
```

Comment:
Add rowNum and attribute here

Reply:
Not all EMX exceptions have a rowNum and attribute, for example "InconsistentPackageStructureException"